model/execution/DeliveryExecutionManagerService.php

```diff
@@ -80,21 +81,43 @@ public function getDeliveryTimer($deliveryExecution)
      * Sets the extra time to a list of delivery executions
      * @param $deliveryExecutions
      * @param null $extraTime
+     * @param null $extendedTime
      * @return array
      */
-    public function setExtraTime($deliveryExecutions, $extraTime = null)
+    public function setExtraTime($deliveryExecutions, $extraTime = null, $extendedTime = null)
```

Comment:
Could you update also `\oat\taoProctoring\helpers\DeliveryHelper::setExtraTime` signature